contracts/src/chainlink/solidity/mocks/TestERC20.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0.
+pragma solidity ^0.6.12;
+
+import "../../../../vendor/starkware-libs/starkgate-contracts/src/starkware/solidity/tokens/ERC20/ERC20.sol";
```

Comment:
Maybe these contracts could belong here ? :
https://github.com/smartcontractkit/chainlink-starknet/tree/develop/contracts/vendor/starkware-libs

Reply:
The contracts under vendor look to not be automatically compile unless if they are used by a contract which is outside vendor folder.